packages/flutter/lib/src/cupertino/radio.dart

```diff
@@ -132,24 +132,22 @@ class CupertinoRadio<T> extends StatefulWidget {
   /// The cursor for a mouse pointer when it enters or is hovering over the
   /// widget.
   ///
-  /// If [mouseCursor] is a [WidgetStateMouseCursor],
-  /// [WidgetStateMouseCursor.resolve] is used for the following [WidgetState]s:
+  /// Resolves in the following states:
   ///
   ///  * [WidgetState.selected].
-  ///  * [WidgetState.hovered].
   ///  * [WidgetState.focused].
   ///  * [WidgetState.disabled].
   ///
-  /// If null, then [SystemMouseCursors.basic] is used when this radio button is disabled.
-  /// When this radio button is enabled, [SystemMouseCursors.click] is used on Web, and
-  /// [SystemMouseCursors.basic] is used on other platforms.
+  /// If null, then [SystemMouseCursors.basic] is used when this radio button is
+  /// disabled. When this radio button is enabled, [SystemMouseCursors.click] is
+  /// used on Web, and [SystemMouseCursors.basic] is used on other platforms.
```

Comment:
Nit: You might just say `defaults to [defaultMouseCursor]` and let people look that up themselves. Or maybe it's more useful for users to see it written out like this, up to you.

Reply:
True. Then I could move these docs to `defaultMouseCursor`.